src/app/components/edit-bar-buttons/edit-bar-buttons.component.html

```diff
@@ -0,0 +1,6 @@
+<div *ngIf="userAuthenticated">
+    <button type="button" *ngIf="edit.view" (click)="edit.switchMode()"><fa-icon [icon]="icons.edit"></fa-icon><span>{{ editBtnText }}</span></button>
+    <button type="button" *ngIf="edit.editing" [disabled]="!allowSaving" (click)="edit.switchMode('save')"><fa-icon [icon]="icons.saveChanges"></fa-icon><span>{{ saveBtnText }}</span></button>
+    <button type="button" class="dangerous" *ngIf="edit.editing && showDeleteButton" (click)="edit.switchMode('delete')"><fa-icon [icon]="icons.delete"></fa-icon><span>{{ deleteBtnText }}</span></button>
+    <button type="button" class="discard" *ngIf="edit.editing" (click)="edit.switchMode('discard')"><fa-icon [icon]="icons.discardChanges"></fa-icon><span>{{ discardBtnText }}</span></button>
+</div>
```

Comment:
```suggestion
</div>

```

Nitpick: Missing new line.